x-pack/plugins/security_solution/public/common/components/exceptions/viewer/helpers.tsx

```diff
@@ -127,38 +125,7 @@ export const getDescriptionListContent = (
 
   return details.reduce<DescriptionListItem[]>((acc, { value, title }) => {
     if (value != null && value.trim() !== '') {
-      const valueElement = (
-        <EuiToolTip content={value} anchorClassName="eventFiltersDescriptionListDescription">
```

Comment:
this just went in 2 days ago: https://github.com/elastic/kibana/pull/125145

I think you maybe need to redo your merge with main and resolve these conflicts, you're deleting a lot of intentional code